goserver/session/session.go

```diff
@@ -0,0 +1,96 @@
+package session
+
+import (
+	"crypto/hmac"
+	"crypto/sha1"
+	"encoding/base64"
+	"flag"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+)
+
+var sessionCookieSecret = flag.String("session_cookie_secret", "", "secret key used to encrypt the session cookie")
+
+const (
+	sessionLifeInSec = 7200
+	cookieLifeInSec  = 7200
+	cookieName       = "session"
+)
+
+// Session stores necessary information for a telehealth session.
+type Session struct {
```

Comment:
Should we just make the fields of Session public? If it's meant to just be a simple struct holding a few values without any logic that would make it simpler to use.

Reply:
Done.